### PR TITLE
Issue 6704: LTS - set writer.rollover.size.bytes.max=134217728 as default

### LINKED ISSUE #6704: LTS - set writer.rollover.size.bytes.max=134217728 as default
**Is your feature request related to a problem? Please describe.**
Current value of 1TB for writer.rollover.size.bytes.max is too high.

**Describe the solution you'd like**
set `writer.rollover.size.bytes.max=134217728` as default
Fix it here https://github.com/pravega/pravega/blob/master/segmentstore/server/src/main/java/io/pravega/segmentstore/server/writer/WriterConfig.java#L42

**Describe alternatives you've considered**
Current value of 1TB is too high

**Additional context**

### PR DESCRIPTION
**Pull request by Amit-Singh40**
Signed-off-by: Amit-Singh40 <amit.singh30@dell.com>

**Change log description**
Issue 6704: LTS - set writer.rollover.size.bytes.max=134217728 as default

**Purpose of the change**  
Fixes #6704

**What the code does**
Changing the MAX_ROLL_OVER_SIZE to 134217728

**How to verify it**
All tests should pass.

### CODE REVIEW

**Comment by sachin-j-joshi** [comment]
Looks good to me.
Please confirm that system tests are green.

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Review by RaulGracia — CHANGES_REQUESTED** [review]
Can we add a test for checking the default value of this?

**Comment by Amit-Singh40** [comment]
> 

Sure @RaulGracia. I will add it. I am just checking whether system test is passing. I will add one more test for that.

**Comment by RaulGracia** [comment]
@Amit-Singh40 adding a default value check in a test should take long, how is the progress on this so we can wrap up this PR?

**Comment by Amit-Singh40** [comment]
> @Amit-Singh40 adding a default value check in a test should take long, how is the progress on this so we can wrap up this PR?

Hi @RaulGracia , I am adding it now. There was some issue with testing the build with ECS storage type so I was working  it on some other story.

**Comment by RaulGracia** [comment]
@Amit-Singh40 you should be responsible for keeping track of the status of your own PRs. This PR is not passing `checkstyle` and it has been like this for days. Please, let's wrap up this PR and avoid any further unnecessary delay.

File: segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Writer Config Test
+ */
+public class WriterConfigTest {
+
+    @Test
+    public void testMaxRolloverSize() {
```
Comment (RaulGracia) [inline segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java:27]: Thanks, that's great. One comment: given that we are not testing any default value for this config class, can we change a bit the test to be something like `testDefaultValues()` and include in the test all the other values apart from max rollover size?

Comment (Amit-Singh40) [reply segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java:27]: Sure Raul, I will do the changes.

File: segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        Assert.assertEquals(60 * 1000L, (long) WriterConfig.FLUSH_TIMEOUT_MILLIS.getDefaultValue());
+        Assert.assertEquals("flush.timeout.milliseconds", WriterConfig.FLUSH_TIMEOUT_MILLIS.getName());
+
+        Assert.assertEquals(15 * 1000L, (long) WriterConfig.ACK_TIMEOUT_MILLIS.getDefaultValue());
+        Assert.assertEquals("ack.timeout.milliseconds", WriterConfig.ACK_TIMEOUT_MILLIS.getName());
+
+        Assert.assertEquals(10 * 1000L, (long) WriterConfig.SHUTDOWN_TIMEOUT_MILLIS.getDefaultValue());
+        Assert.assertEquals("shutDown.timeout.milliseconds", WriterConfig.SHUTDOWN_TIMEOUT_MILLIS.getName());
+
+        Assert.assertEquals(134217728L, (long) WriterConfig.MAX_ROLLOVER_SIZE.getDefaultValue());
```
Comment (RaulGracia) [inline segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java:64]: Is this being duplicated with the first one?

**Review by RaulGracia — APPROVED** [review]
LGTM, just one minor comment.

File: segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java (hunk 2 above)
Comment (shshashwat) [reply segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java:64]: yes, it looks like duplicate with the first one at line no 28 and 29

Comment (Amit-Singh40) [reply segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java:64]: Sorry, missed it. Let me remove it.

Comment (Amit-Singh40) [reply segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java]: @RaulGracia removed it.

**Pull request merged**